TicketsDemo.Data/OptionsForCalculationPrice/TeaCoffeeBedParametrs.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using TicketsDemo.Data.Entities;
+
+namespace TicketsDemo.Data.OptionsForCalculationPrice
+{
+    public struct TeaCoffeeBedParametrs
+    {
+        public Ticket ticket { get; set; }
+        public PlaceInRun placeInRun { get; set; }
```

Comment:
please don't mess with  common C# naming conventions. Properties should be defined in PascalCase